content/blog/blog-2020-07-30-introducing-operators-index.md

```diff
@@ -0,0 +1,85 @@
+---
+date: 2020-07-30
+---
+
+# Introducing the KUDO Operators Index
+
+The default repository when installing operators with KUDO is the community repository. Adding new operators to this repository has been greatly simplified with a new index of operators and a new community repository URL that is used starting with KUDO 0.16.
+
+<!-- more -->
+
+Adding new operators to the community repository was traditionally done by adding the operator packages as a folder in the [operators](https://github.com/kudobuilder/operators) project. While this worked well for smaller operators, it created challenges for large operators, for example, [KUDO Cassandra](https://github.com/mesosphere/kudo-cassandra-operator). Larger operators are developed as separate projects in their own git repository. For new releases, the operator package would have to be copied over to the _operators_ project. In case of bugs in the operator, is the source of truth now the origin project or the folder in the _operators_ project?
+
+To avoid these and other challenges, the new [Operators Index](https://github.com/kudobuilder/operators-index) provides a simple way to reference operator packages located in external projects. Instead of copying an operator package, operator developers reference the URL of a git repository and the operator packages included in that git repository.
+
+With these references, there's a one-to-one mapping of operator packages included by a git repository and the respective packages in KUDO's community repository. Every entry in the _Operators Index_ corresponds to a package in the community repository.
+
+## Adding operator packages to the _Operators Index_
+
+To add an operator package, create a pull request that adds or updates an index entry in the [operators](https://github.com/kudobuilder/operators-index/tree/main/operators) folder of the [Operators Index](https://github.com/kudobuilder/operators-index). Each entry in the index references versions of an operator package.
+By creating a pull request that adds or updates an entry, the referenced operator packages are validated as part of a CI job. This is similar to running `kubectl kudo package verify` on the operator package. Additional checks confirm that the metadata of the entry matches the respective metadata of the operator package.
```

Comment:
Should we mention that warnings are treated as errors here? They are, right?

Reply:
That's the plan, but they aren't as of right now because of some validation bugs in KUDO that will be fixed in 0.16.